scripts/setup-system-files.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# setup raspbian / raspberry pi configuration files
+# /boot/cmdline.txt and
+# /boot/config.txt
+. $COMPANION_DIR/scripts/bash-helpers.sh
+
+run_step echo "" > /etc/wpa_supplicant/wpa_supplicant.config
```

Comment:
the file must have some valid content for `wpa_cli scan_results` to work correctly.